dhall/src/Dhall/Syntax.hs

```diff
@@ -211,9 +215,7 @@ newtype DhallDouble = DhallDouble { getDhallDouble :: Double }
 --
 -- This instance is also consistent with with the binary encoding of Dhall @Double@s:
 --
--- >>> toBytes n = Dhall.Binary.encodeExpression (DoubleLit n :: Expr Void Import)
---
--- prop> \a b -> (a == b) == (toBytes a == toBytes b)
```

Comment:
@quasicomputational I'm under the impression that `stack test` already exercises this property test. At least it does fail when I apply this change:

```diff
--- prop> \a b -> (a == b) == (toBytes a == toBytes b)
+-- prop> \a b -> (a == b) == (toBytes a == toBytes c)
```

```
/home/simon/src/dhall-haskell/dhall/src/Dhall/Syntax.hs:222: failure in expression `\a b -> (a == b) == (toBytes a == toBytes c)'
*** Failed! Falsifiable (after 77 tests):
DhallDouble {getDhallDouble = -9.223372036854775e18}
DhallDouble {getDhallDouble = -9.223372036854775e18}
DhallDouble {getDhallDouble = 7.777191964028747}

Examples: 201  Tried: 201  Errors: 0  Failures: 1
```

Although I have _no_ idea where it found that `c`. This is so confusing!